start-tests.sh

```
@@ -170,10 +170,17 @@ log_color "green" "Testing with ACM Version": "$CYPRESS_ACM_VERSION\n"
 installNamespace=`oc get subscriptions.operators.coreos.com --all-namespaces | grep advanced-cluster-management | awk '{print $1}'`
 
 # Search for managed clusters.
-MANAGED_CLUSTERS=($(oc get managedclusters -o custom-columns='name:.metadata.name' --no-headers))
+MANAGED_CLUSTERS=($(oc get managedclusters -l '!local-cluster' -o custom-columns='name:.metadata.name' --no-headers))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)